src/Confluent.SchemaRegistry.Serdes.Json/JsonSerializerConfig.cs

```diff
@@ -64,13 +64,21 @@ public static class PropertyNames
             /// <summary>
             ///     Specifies whether or not the JSON serializer should use the latest subject
             ///     version for serialization.
-            ///     WARNING: There is no check that the latest schema is backwards compatible
-            ///     with the schema of the object being serialized.
             ///
             ///     default: false
             /// </summary>
             public const string UseLatestVersion = "json.serializer.use.latest.version";
 
+            /// <summary>
```

Comment:
Maybe: "Specifies whether or not the JSON serializer should check compatibility with the latest schema if `use.latest.version` is set to true."

note that the compatibility check in the java case is backwards, because that is what is provided by the client library. however we are using schema registry to do the check, and that may be forwards, backwards or full, depending on what is configured.

Reply:
True. Making changes